lib/trino-parquet/src/main/java/io/trino/parquet/hadoop/metadata/BlockMetaData.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.parquet.hadoop.metadata;
+package io.trino.parquet.hadoop.metadata;
```

Comment:
Remove `hadoop` from the name so make this `io.trino.parquet.metadata`

Same for the rest

Reply:
Let's also rename these classes so that `Metadata` is one word